website/docs/r/monitor_cloud_account.md

```diff
@@ -15,18 +15,39 @@ Creates a Sysdig Monitor Cloud Account for monitoring cloud resources.
 ## Example Usage
 
 ```terraform
+// GCP example
 resource "sysdig_monitor_cloud_account" "sample" {
   cloud_provider = "GCP"
   integration_type = "API"
   account_id = "gcp_project_id"
 }
+
+// AWS example with role delegation
+resource "sysdig_monitor_cloud_account" "sample" {
+  cloud_provider = "AWS"
+  integration_type = "Metrics Streams"
+  account_id = "123412341234"
+  role_name = "SysdigTestRole"
+}
+
+// AWS example with secret key
+resource "sysdig_monitor_cloud_account" "sample" {
+  cloud_provider = "AWS"
+  integration_type = "Metrics Streams"
+  account_id = "123412341234"
+  secret_key = "Xxx5XX2xXx/Xxxx+xxXxXXxXxXxxXXxxxXXxXxXx"
+  access_key_id = "XXXXX33XXXX3XX3XXX7X"
+}
 ```
 
 ## Argument Reference
 
-* `cloud_provider` - (Required) Cloud platform that will be monitored. Only `GCP` is currently supported.
-* `integration_type` - (Required) Type of cloud integration. Only `API` is currently supported.
-* `account_id` - (Required) The GCP project id for the project that will be monitored.
+* `cloud_provider` - (Required) Cloud platform that will be monitored. Only `GCP` and `AWS` are currently supported.
+* `integration_type` - (Required) Type of cloud integration. Only `API` and `Metrics Streams` are currently supported (`Metrics Streams` only for `AWS`).
+* `account_id` - (Required) The GCP project id for the project that will be monitored and Account ID itself for AWS.
+* `role_name` - (Optional) The role name tha will take the permissions over some resources in AWS
+* `secret_key` - (Optional) The the secret key for a AWS connection
```

Comment:
also states here that the secret and access key must be both provided to use this auth mode